cocotb/share/lib/fli/FliCbHdl.cpp

```diff
@@ -26,6 +26,10 @@
 ******************************************************************************/
 
 #include "FliImpl.h"
+#include "mti.h"
+#undef DLLEXPORT  // redefinition in tcl.h causes warning
```

Comment:
Can you clarify where the original definition came from too?

Reply:
`gpi.h`, but it isn't used anywhere in the project, so we should probably remove it.

Reply:
Removed it in a separate commit.